site/en/docs/extensions/mv3/hosting/index.md

```diff
@@ -1,69 +1,54 @@
 ---
 layout: "layouts/doc-post.njk"
-title: "Chrome Web Store"
+title: "Extension hosting"
 date: 2012-09-18
-updated: 2018-06-12
+updated: 2021-12-10
 description: >
-  How to host your extension in the Chrome Web Store and update an extension
-  that's hosted in the Chrome Web Store.
+  How to host your Chrome extension.
 ---
 
-{% include 'partials/extensions/mv2page-in-mv3.md' %}
+<!--
+Reframe this to focus explicitly on hosting.
 
-Most extensions are hosted in the [Chrome Web Store][1] to best [protect users from malicious
-extensions][2].
+2 options:
+- CWS
+- Self-hosting
 
-## Hosting {: #hosting }
+CWS is by far the most common
 
-All extensions are distributed to users as a special ZIP file with a `.crx` suffix. Extensions
-hosted in the [Chrome Web Store][3] are uploaded through the [Developer Dashboard][4] as `.zip`
-files. The [publishing][5] process automatically converts the `.zip` into a `.crx` file.
+note that during development you can also load unpacked.
+-->
 
-There are three exceptions to the Chrome Web Store hosting rule:
+There are several different ways to initiate the installation of a Chrome extension, but there are
+only two officially supported distribution mechanisms.
 
-1.  Extensions that are distributed through the [enterprise policy][6].
-2.  Unpacked extension directories from a local machine while in [developer mode][7].
-3.  [Linux installation][8].
+[Chrome Web Store][cws-about]
 
-[Read more about the hosting policy][9].
+: Chrome Web Store is an online marketplace for Chrome extensions and themes. Developers who
+  register with the Chrome Web Store can publish their extensions and make them available to users
+  across the world. Only extensions hosted on and signed by the Chrome Web Store can be directly
+  installed by end users\*. See [Publish in the Chrome Web Store][cws-publish] and [Enterprise
+  publishing options][cws-enterprise] for more information about how to publish on Chrome Web Store.
 
-## Updating {: #updating }
+Self-hosting
 
-The Chrome Browser periodically checks for new versions of installed extensions and updates them
-without user intervention.
+: is the practice of hosting an extension outside of the Chrome Web Store. This option is used in
```

Comment:
```suggestion
: Self hosting is the practice of hosting an extension outside of the Chrome Web Store. This option is used in
```

Reply:
This may violate the best practice to omit needless words. This follows another best practice: "express coordinate ideas in similar form" (aka "parallelism", Strunk and White, rule II 19).